solr/core/src/java/org/apache/solr/packagemanager/PackageUtils.java

```diff
@@ -155,9 +155,12 @@ public static String getFileFromJarsAsString(List<Path> jars, String filename) {
   }
 
   /** Returns JSON string from a given Solr URL */
-  public static String getJsonStringFromUrl(SolrClient client, String path, SolrParams params) {
+  public static String getJsonStringFromUrl(
```

Comment:
I wonder if it would be simpler to just have "getJsonStringFromCollection" and "getJsonStringFromSolr" or something?   After all, it's not a generic "getJsonStringFromUrl" where I can pass in any URL, despite what the name of the method appears to suggest...!    Then we could skip passing a magic boolean around...

Reply:
Yeah, the 'magic boolean' is less than ideal for sure.  Done!